src/overlays/actors/ovl_En_Gm/z_en_gm.h

```diff
@@ -6,9 +6,21 @@
 
 struct EnGm;
 
+typedef void (*EnGmActionFunc)(struct EnGm*, GlobalContext*);
+
 typedef struct EnGm {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x184];
+    /* 0x014C */ SkelAnime skelAnime;
+    /* 0x0190 */ Vec3s limbDrawTable[18];
+    /* 0x01FC */ Vec3s transitionDrawTable[18];
+    /* 0x0268 */ EnGmActionFunc actionFunc;
+    /* 0x026C */ EnGmActionFunc subActionFunc;
```

Comment:
Based on the usage, my guess is these are supposed to be differentiated as update modes and actions. Probably good to hear the opinion of others on this case tho.
```suggestion
    /* 0x0268 */ EnGmUpdateFunc updateFunc;
    /* 0x026C */ EnGmActionFunc actionFunc;
```